scripts/lint.sh

```diff
@@ -14,7 +14,7 @@ function lint_scripts() {
 
 function lint_jvm_language_code() {
     ./gradlew ktlintFormat
-    ./gradlew check -x test
+    ./gradlew clean check -x test
```

Comment:
This forces an entire rebuild though. It's going to waste a lot of time.

It feels like there should be a better way to achieve this.

Reply:
If you want to check all the code, I think you have to build all of it.

The current script before this fix is broken, because it only checks the code that hasn't already been built locally.

Reply:
Where did we build everything before the "gradle clean check"?

Reply:
Well, whenever we do what the "gradle clean" cleans.

Reply:
That could have been any other build activity unrelated to the current gradle check run. And that's why the results were inconsistent.